packages/manager/modules/office/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@ovh-ux/office",
+  "version": "7.0.8",
```

Comment:
Since you are changing the package name I think we can init the semver to v1.0.0.